src/shared/tests.h

```diff
@@ -3,6 +3,27 @@
 
 #include <stdbool.h>
 
+#include "sd-daemon.h"
+
+#include "string-util.h"
+
+#define TEST_REQ_RUNNING_SYSTEMD(x)                                 \
+        if (sd_booted() > 0) {                                      \
+                x;                                                  \
+        } else {                                                    \
+                printf("systemd not booted skipping '%s'\n", #x);   \
+        }
+
+#define MANAGER_SKIP_TEST(r)                                    \
+        IN_SET(r,                                               \
+               -EPERM,                                          \
+               -EACCES,                                         \
+               -EADDRINUSE,                                     \
+               -EHOSTDOWN,                                      \
+               -ENOENT,                                         \
+               -ENOMEDIUM /* cannot determine cgroup */         \
+               )
```

Comment:
hmm, it would appear more natural to me if the closing `)` would be indented to 8ch instead of 16ch, but i guess this doesn't matter